packages/integration/package.json

```diff
@@ -3,7 +3,6 @@
   "version": "1.2.0",
   "description": "Zero-runtime Stylesheets-in-TypeScript",
   "main": "dist/vanilla-extract-integration.cjs.js",
-  "module": "dist/vanilla-extract-integration.esm.js",
```

Comment:
Decided to only export a CJS version of the integration as there was no reason to have ESM